src/Clip.cpp

```diff
@@ -907,6 +935,7 @@ Json::Value Clip::JsonValue() const {
 	root["anchor"] = anchor;
 	root["display"] = display;
 	root["mixing"] = mixing;
+	root["blend"] = blend;
```

Comment:
So with a Keyframe property, like the others you'd want to set this to the  JSON representation of its current data:
```suggestion
	root["blend"] = blend.JsonValue();
```